ElectronClient/app.js

```diff
@@ -662,9 +662,21 @@ class Application extends BaseApplication {
 				message.push(`\n${gitInfo}`);
 				console.info(gitInfo);
 			}
-			bridge().showInfoMessageBox(message.join('\n'), {
+			const text = message.join('\n');
+
+			const copyLabel = _('Copy');
+			// The following hack helps to position the OK button approximately in the middle
+			// of the dialog box (if the copyLabel is not longer than 14 characters)
+			const spacer = ' '.repeat(Math.max(0, Math.trunc((14 - copyLabel.length) / 2)));
+
+			const copyToClipboard = bridge().showConfirmMessageBox(text, {
 				icon: `${bridge().electronApp().buildDir()}/icons/128x128.png`,
+				buttons: [spacer + copyLabel + spacer, _('OK')],
+				defaultId: 1,
 			});
+			if (copyToClipboard) {
+				clipboard.writeText(text);
+			}
```

Comment:
Rather than overriding the behaviour of the `showConfirmMessageBox`, could you create a new bridge function `showMessageBox`, which calls `showMessageBox_` and that can be used for special cases like this one, and that would return the exact buttonId that was clicked on?

Reply:
Sure, will do. I'll do this in the afternoon.